brightray/BUILD.gn

```diff
@@ -0,0 +1,42 @@
+import("//build/config/ui.gni")
+
+filenames_gypi = exec_script(
+  "//build/gypi_to_gn.py",
+  [ rebase_path("filenames.gypi") ],
+  "scope",
+  [ "filenames.gypi" ]
+)
+
+static_library("brightray") {
+  deps = [
+    "//base",
+    "//components/network_session_configurator/common",
+    "//components/prefs",
+    "//content/public/browser",
+    "//content/shell:resources",
+    "//net:net_with_v8",
+    "//skia",
+    "//ui/views",
+  ]
+
+  include_dirs = [
+    "..",
+  ]
+
+  defines = [
+    "DISABLE_NACL=1",
+  ]
+
+  extra_source_filters = []
+  if (is_mac) {
+    extra_source_filters += [
+      "*_views.cc",
+      "*_views.h",
+      "*\bviews/*",
```

Comment:
Why do we need those on Mac?

Reply:
Because the code in there doesn't build on Mac, and also isn't used on Mac. In gyp-land, those filenames are filtered out by `electron/brightray/filename_rules.gypi`, but GN's filename filter list is shorter (see https://cs.chromium.org/chromium/src/build/config/BUILDCONFIG.gn?q=buildconfig&sq=package:chromium&dr&l=408)